src/rules/__tests__/no-jasmine-globals.test.ts

```diff
@@ -53,6 +53,10 @@ ruleTester.run('no-jasmine-globals', rule, {
       errors: [{ messageId: 'illegalJasmine', column: 1, line: 1 }],
       output: 'jest.setTimeout(5000);',
     },
+    {
+      code: 'jasmine.DEFAULT_TIMEOUT_INTERVAL = function() {}',
```

Comment:
is this valid jasmine code?

Reply:
I'm happy to warn on it either way

Reply:
I don't think so, but it's technically something that could be done. Eitherway, it triggers the previously uncovered branch :)